entrypoint.sh

```diff
@@ -0,0 +1,168 @@
+    # https://stackoverflow.com/questions/64194634/why-pip-freeze-returns-some-gibberish-instead-of-package-version
+    if [ "${pythonLocation:-}" != "" ]; then
+        SAFETY_OS_DESCRIPTION="${SAFETY_OS_DESCRIPTION} env_pythonLocation"
+
+        docker run --rm --privileged --pid=host justincormack/nsenter1 "${pythonLocation}/bin/python" -m pip list --format=freeze > /tmp/requirements.txt
+    else
+        SAFETY_OS_DESCRIPTION="${SAFETY_OS_DESCRIPTION} env_root"
+
+        docker run --rm --privileged --pid=host justincormack/nsenter1 /bin/sh -c "pip list --format=freeze" > /tmp/requirements.txt
+    fi
+    SAFETY_ACTION_REQUIREMENTS="/tmp/requirements.txt"
+elif [ "${SAFETY_ACTION_SCAN}" = "file" ]; then
+    if [[ "${SAFETY_ACTION_REQUIREMENTS}" == "" ]]; then
+        SAFETY_OS_DESCRIPTION="${SAFETY_OS_DESCRIPTION} file_path_scan"
+
+        if [ -e "poetry.lock" ]; then
+            SAFETY_OS_DESCRIPTION="${SAFETY_OS_DESCRIPTION} file_path_scan_poetry"
+            SAFETY_ACTION_REQUIREMENTS="$(pwd)/poetry.lock"
+        elif [ -e "Pipfile.lock" ]; then
+            SAFETY_OS_DESCRIPTION="${SAFETY_OS_DESCRIPTION} file_path_scan_pipfile"
+            SAFETY_ACTION_REQUIREMENTS="$(pwd)/Pipfile.lock"
+        elif [ -e "requirements.txt" ]; then
+            SAFETY_OS_DESCRIPTION="${SAFETY_OS_DESCRIPTION} file_path_scan_requirementstxt"
+            SAFETY_ACTION_REQUIREMENTS="$(pwd)/requirements.txt"
+        else
+            echo "[Safety Action] Could not autodetect a poetry.lock / Pipfile.lock / requirements.txt automatically. Try set the requirements variables in the action configuration." 1>&2
+            exit 1
+        fi
+
+        echo "[Safety Action] Autodetecting Requirements File: ${SAFETY_ACTION_REQUIREMENTS}" 1>&2
+    else
+        SAFETY_OS_DESCRIPTION="${SAFETY_OS_DESCRIPTION} file_path_specified"
+    fi
+
+    echo "[Safety Action] Scanning Requirements File: ${SAFETY_ACTION_REQUIREMENTS}" 1>&2
+
+    if [[ "$(basename ${SAFETY_ACTION_REQUIREMENTS})" == "Pipfile.lock" ]]; then
+        SAFETY_OS_DESCRIPTION="${SAFETY_OS_DESCRIPTION} file_pipfile_converted"
+
+        cd "$(dirname ${SAFETY_ACTION_REQUIREMENTS})"
+        pipenv requirements > /tmp/requirements.txt
+        echo "[Safety Action] Converted ${SAFETY_ACTION_REQUIREMENTS} to /tmp/requirements.txt using pipenv." 1>&2
+        SAFETY_ACTION_REQUIREMENTS="/tmp/requirements.txt"
+    elif [[ $(basename ${SAFETY_ACTION_REQUIREMENTS}) == "poetry.lock" ]]; then
+        SAFETY_OS_DESCRIPTION="${SAFETY_OS_DESCRIPTION} file_poetry_converted"
+
+        cd "$(dirname ${SAFETY_ACTION_REQUIREMENTS})"
+        poetry export -f requirements.txt --without-hashes > /tmp/requirements.txt
+        echo "[Safety Action] Converted ${SAFETY_ACTION_REQUIREMENTS} to /tmp/requirements.txt using poetry." 1>&2
+        SAFETY_ACTION_REQUIREMENTS="/tmp/requirements.txt"
+    fi
+
+    if [[ "${SAFETY_ACTION_CONTINUE_ON_ERROR}" != "" ]]; then
```

Comment:
@cb22, a whitespace may change the behavior in the action, I think we could improve the check here.

Reply:
@yeisonvargasf yes true. We could make it accept (yes|true), case insensitive?

Reply:
Agree!